plugins/polio/js/src/domains/Campaigns/Rounds/RoundsForm.js

```diff
@@ -203,7 +203,7 @@ export const RoundsForm = () => {
                 )}
             </Box>
             {currentRoundNumber !== undefined && (
-                <Box mt={2}>
+                <Box mt={2} style={{ width: '100%' }}>
```

Comment:
you can use width props directly 
```suggestion
                <Box mt={2} width="100%" >
```